getting_started/step_by_step/scenes_and_nodes.rst

```diff
@@ -6,63 +6,64 @@ Scenes and nodes
 Introduction
 ------------
 
-.. image:: img/chef.png
+In this chapter we're going to handle two main concepts: Scenes and Nodes.
+These are fundamental basic concepts and it's absolutely necessary to
+understand what these concepts are when working in Godot. A node is any
+object that you want to create (for example a spaceship) and a scene
+is a collection of nodes (a spaceship, its sprite and a sound).
 
-Imagine for a second that you are not a game developer anymore. Instead,
-you're a chef! Change your hipster outfit for a toque and a double
-breasted jacket. Now, instead of making games, you create new and
-delicious recipes for your guests.
-
-So, how does a chef create a recipe? Recipes are divided into two
-sections: the first is the ingredients and the second is the
-instructions to prepare it. This way, anyone can follow the recipe and
-savor your magnificent creation.
-
-Making games in Godot feels pretty much the same way. Using the engine
-feels like being in a kitchen. In this kitchen, *nodes* are like a
-refrigerator full of fresh ingredients with which to cook.
```

Comment:
We're proud of our documentation having a few light-hearted sections (which are actually very rare), and the overwhelming majority of users who do comment about it seem to enjoy it too.

So this is not going away. And definitely not without discussion in a PR with 45 commits.

Reply:
There should only be 15 commits (the ones pushed 2020-02-05). Not sure why the previous 30 got added to this PR since they were already merged with your master and are no longer active in my fork, they contain grammar fixes. 

In my opinion verbose text doesn't belong in a tutorial and/or documentation. I understand the purpose is to add something light-hearted but for me it just got in the way. I'm there to learn an already complex concept with a technical explanation so I can start making games. Even though it is light-hearted (even a little whimsical) it only added another layer to unpack instead of hitting premise after premise like a numbered list, which is what I think an effective tutorial should do. It added 3 paragraphs to the tutorial that I had to read in order to decide I could ignore them completely. 

With that said, you're the boss, and I'll make sure it's in there as soon as I find the time.